ConcurrencyHomework/src/concurrency/Main.java

```diff
@@ -0,0 +1,64 @@
+package concurrency;
+
+import java.io.*;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+
+        List<String> personInfo = new ArrayList<>();
+
+        //read information from the file 1
+        try (BufferedReader file1 = new BufferedReader(new InputStreamReader(new FileInputStream("D:\\JAVA EX\\scoala-informala-homework\\ConcurrencyHomework\\file1.txt")))) {
+            String s = null;
+            while ((s = file1.readLine()) != null) {
+                personInfo.add(s);
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        // read information from the second file 2
+        try (BufferedReader file2 = new BufferedReader(new InputStreamReader(new FileInputStream("D:\\JAVA EX\\scoala-informala-homework\\ConcurrencyHomework\\file2.txt")))) {
+            String s = null;
+            while ((s = file2.readLine()) != null) {
+                personInfo.add(s);
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+
+        DateTimeFormatter format = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+
+        List<Person> femaleBirthday = new ArrayList<>();
+
+        for (int i = 0; i < personInfo.size(); i++) {
```

Comment:
Logica din acest for ar fi trebuit sparta in doua responsabilitati diferite: creare de obiecte `Person` folosind datele citite din fisier si filtrarea tuturor obiectelog cu `Gender.FEMALE` si data nasterii in data curenta. Astfel codul era mai organizat, usor de inteles, testabil.